src/async_impl/client.rs

```diff
@@ -150,47 +152,33 @@ impl ClientBuilder {
     /// site will be trusted for use from any other. This introduces a
     /// significant vulnerability to man-in-the-middle attacks.
     #[inline]
-    pub fn danger_disable_hostname_verification(&mut self) -> &mut ClientBuilder {
-
+    pub fn danger_disable_hostname_verification(&mut self, accept_invalid_hostname: bool) -> &mut ClientBuilder {
```

Comment:
The names may cause some confusion, since the method name is to *disable*, and the argument name is to *accept*. I eventually realized that they were "disable verification" and "accept non verified", meaning the same thing, but I was quite confused at first!

Reply:
Oh yeah, `native-tls` uses `danger_accept_invalid_certs(accept_invalid_certs: bool)`. I can do that, and make sure that the disable/accept verbiage is consistent.